### PR TITLE
Add first version of script for D resonances

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: resonances/config_fit_Ds2starplus.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+input: outputs/Ds2starplus/Dplus_K0S_HM_centsel.parquet.gzip
+output_dir: outputs/Ds2starplus
+pdg_reso: 435
+
+fit_config:
+  pt_mins: [2.]
+  pt_maxs: [20.]
+  mass_mins: [0.55, 0.55]
+  mass_maxs: [0.85, 0.85]
```
Comment (fcatalan92) [inline resonances/config_fit_Ds2starplus.yml:9]:
```suggestion
  mass_mins: [0.55]
  mass_maxs: [0.85]
```
This looks like a remnant of a multi-bin fit.

File: resonances/config_fit_Ds1plus.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+input: outputs/Ds1plus/Dstar_K0S_HM_centsel.parquet.gzip
+output_dir: outputs/Ds1plus
+pdg_reso: 10433
+
+fit_config:
+  pt_mins: [2.]
+  pt_maxs: [20.]
+  mass_mins: [0.0, 0.0]
+  mass_maxs: [0.6, 0.6]
```
Comment (fcatalan92) [inline resonances/config_fit_Ds1plus.yml:9]:
```suggestion
  mass_mins: [0.0]
  mass_maxs: [0.6]
```
This looks like a remnant of a multi-bin fit.

**Review by fcatalan92 — APPROVED** [review]
Hi @fgrosa, thanks for the implementations. My comments are mostly minor.

File: resonances/project_DV0reso.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
+import os
+import argparse
+import yaml
+import matplotlib.pyplot as plt
+import uproot
+from hist import Hist
+from flarefly.data_handler import DataHandler
+from flarefly.fitter import F2MassFitter
+from particle import Particle
```
[the remark this answers is not in the document]
Comment (fgrosa) [reply resonances/project_DV0reso.py:9]: It should be in the requirements of `zfit` (https://github.com/zfit/zfit/blob/develop/requirements.txt), which is in the requirements of `flarefly`, so it should be ok (unless you tried and did not work)

File: resonances/project_DV0reso.py [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+        fitter_v0 = F2MassFitter(data_hdl_v0, ["gaussian"], ["chebpol1"], "Lambda_ptint")
+        fitter_v0.set_signal_initpar(0, "sigma", 0.0001)
+
+    fitter_v0.set_particle_mass(0, pdg_id=pdg_v0)
+    fitter_v0.mass_zfit()
+    fig_v0 = fitter_v0.plot_mass_fit(style="ATLAS", bins=200, axis_title=label_mass_v0)
+    mean_v0, _ = fitter_v0.get_mass(0)
+    frac_gaus_v0 = fitter_v0.get_signal_parameter(0, "frac1")[0]
+    width_v0 = frac_gaus_v0*fitter_v0.get_signal_parameter(0, "sigma1")[0] + \
+        (1-frac_gaus_v0)*fitter_v0.get_signal_parameter(0, "sigma2")[0]
```
[the remark this answers is not in the document]
Comment (fgrosa) [reply resonances/project_DV0reso.py:111]: No, I will fix it now

File: resonances/project_DV0reso.py [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+import uproot
+from hist import Hist
+from flarefly.data_handler import DataHandler
+from flarefly.fitter import F2MassFitter
+from particle import Particle
+
+def project(config, trigger, pdg_d, pdg_v0):
+    """
+    function for raw yield computation
+    """
```
[the remark this answers is not in the document]
Comment (fgrosa) [reply resonances/project_DV0reso.py]: Done

File: resonances/extract_rawyield_DV0reso.py [hunk 6]
```diff
@@ -0,0 +1,147 @@
+"""
+Script for raw yield extraction
+"""
```
[the remark this answers is not in the document]
Comment (fgrosa) [reply resonances/extract_rawyield_DV0reso.py:3]: Done

**Pull request merged**